### PR TITLE
Revert "Improves Boxstation medbay"

### PR DESCRIPTION
**Pull request by Wilchenx**
Its awful

straight line maintenance is the worst part.

Link to webmap so you can see it yourself 
https://affectedarc07.github.io/SS13WebMap/BeeStation/BoxStation/

### CODE REVIEW

**Comment by Wilchenx** [comment]
image
image

**Comment by Wilchenx** [comment]
image

I'm at loss at the absolute state of boxstation maintenance.

**Comment by St0rmC4st3r** [comment]
I don't like the straight maintenance either, but I don't really see a way to fix it. As for reverting to the madhouse it was before - no, thanks, I'm against it (obviously).

**Comment by St0rmC4st3r** [comment]
Also, elaborate why in your opinion is it awful?

**Comment by Wilchenx** [comment]
image
this sleeper blocked by crate and glass table
image
this could've been piped more efficiently
image
why is the break room before a treatement room?
image
whats the point of multiple treatement rooms
image
you could've done something with this space but it still has no purpose and will never be used
image
3x3 random maint room will usually roll god awful rooms that won't be easily passable

**Comment by qwertyquerty** [comment]
I would say fix the issues instead of reverting

**Comment by ike709** [comment]
Man this was a pretty stinky change yeah.

**Comment by 4rchibald** [comment]
Ok, issues like boxes blocking sleepers call for fix not reverting just like @qwertyquerty pointed out but I can only assume they were just few example and whole PR doesn't really work for many people. **Could someone who plays medic tell us more? What are the issues with current medbay and what were the issues with old one?** 
I kinda feel like trying mapping again but I'm not really sure what you guys are looking for to make your shift run smooth. I proposed few changes in #2172 but that was just my thinking and i have almost none medical experience. If someone could explain here what they like and don't like about old and current medbay, what would they like to see and what they want changed or gone I could try reworking it...I was about to do it anyway since I'm triggered by some distro pipes not being visible in maints around medbay now

**Comment by St0rmC4st3r** [comment]
@Wilchenx, your points and questions first:

>This sleeper blocked by crate and glass table.

Not perfect, not horrible. Should've changed the places with the shower, I guess.

>This could've been piped more efficiently.

It could've. But I didn't like that it made the room somewhat empty and/or having pockets of space.

>Why is the break room before a treatement room?

It's not a separate room, though. Maybe I could've put some supplies like mask boxes or something, but there were already tables for that in an area that suited them better, and people wanted a break area, so I couldn't come up with anything better.

>Whats the point of multiple treatement rooms?

The motivation behind this design was to stop random people from running around the medbay for no reason and make treatment centres further apart to separate patients by severity of damage. The northern room is intended for patients that are just slightly injured and/or first aid to those who are severely injured. The southern room is a mix of a treatment room and post-operation recovery room. By separating them we don't have assistants with mild bruises interrupting a doctor that is tending to someone actively dying.

>You could've done something with this space but it still has no purpose and will never be used.

In my eyes, these rooms are used for times when you need to performed prolonged healing (yes, that happens too) or when you need extra privacy in the last moments of the patient's life. If you have any ideas on what to put there - you are free to do so.

>3x3 random maint room will usually roll god awful rooms that won't be easily passable.

Fair point. I have put it there because it was there originally. That's it. 

@4rchibald now you.

>Could someone who plays medic tell us more?

This will be definitely biased since I made the original changes, but I was making them as someone who plays as a doctor most of the time. My issues with the old medbay were that, as I have pointed earlier, people running around, swapping places with doctors, "helping" with healing which results in damages and doctors stealing the patients because in a tight room nobody knows who is treating who and which patient is waiting for what treatment. 

>Distro pipes not being visible in maints around medbay now.

Yes, my bad. I made a followup PR that should have fixed most of them but there could indeed be a problem somewhere.

In conclusion: we need more data from different people on the issues of the new medbay. Apart from the issues @Wilchenx has described, some of which were conscious design decisions and the other - minor oversights, I feel that this revert is motivated by "New bad, old good". I honestly believe that the new medbay has more advantages than flaws, which do exist, but none of them are severe enough to justify completely reverting the changes.

**Comment by Wilchenx** [comment]
Some additional remarks, number of airlocks and windows leading to the hallway has been greatly reduced making the hallway feel extremely empty, it's just one straight wall that really clashes with everything else

**Comment by St0rmC4st3r** [comment]
>Some additional remarks, number of airlocks and windows leading to the hallway has been greatly reduced making the hallway feel extremely empty, it's just one straight wall that really clashes with everything else

I had the second airlocks at some point, but decided that it conflicts with the design idea when during a testmerge I saw them being set to emergency access, thus defeating the entire point.

**Comment by Wilchenx** [comment]
https://en.wikipedia.org/wiki/Desire_path

There was a reason it was set to emergency access.

**Comment by VictorVow** [comment]
Overall it's a change for the better not warranting a revert based on the treatment rooms and plumbing room alone. Needs a few fixes however.

**Comment by eLeCtrOssSnake** [comment]
No, new medbay is shit and should be deleted and forgotten.

**Comment by zeskorion** [comment]
note that the maint room passability issues have been fixed

**Comment by Blueturbo47** [comment]
This was voted on, 51 up to 5 down, if you think your gonna revert it your vote to revert isn't going to work, instead make a pr fixing the issues not reverting them.

**Comment by zeskorion** [comment]
conflicts, try making a pr to improve instead of revert
